assets/css/style.css

```diff
@@ -415,8 +413,216 @@ footer a {
   margin-bottom: 50px;
 }
 
-footer {
-  display: flex;
-  justify-content: center;
-  justify-items: center;
+.display {
+  display: block;
+}
+
+.show {
+  display: none;
+}
+
+@media only screen and (min-width: 768px) {
```

Comment:
- [ ] Kindly after finishing implementing all changes on the desktop version. Let's take a step back and make everything on our mobile version aligned correctly as on the provided Figma design. :wink: Please make sure that everything looks exactly like the provided design to make sure that our app is fully responsive :100:

Reply:
fixed